### PR TITLE
Add code coverage step to circle CI pipeline, add coveralls dependency

### PR DESCRIPTION
**Pull request by marcciosilva**
- Add coverage step to circleci pipeline (remove Bridge's unit testing phase, since the coverage one already runs those tests)
- Add coveralls' coverage badge to Bridge's README.md
- Fix typo in the Bridge's `deployIntegrationTest` npm script

### CODE REVIEW

**Comment by RafilxTenfen** [comment]
Nice! :D

**Review by pmprete — APPROVED** [review]
Good work Marccio! but i think we need to fix the time it takes to the tests before merging this.
Coverage takes more resources than normal tests, and it's getting to many errors from timeout at the moment, so i would wait we fix that before merging this one

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-version: 2
+version: 2.1
+orbs:
+  coveralls: coveralls/coveralls@1.0.6
```
Comment (pmprete) [inline .circleci/config.yml:3]: awsome did not know about this 💯

**Review by pmprete — APPROVED** [review]
Nice!

**Pull request merged**